app/api/v1.1/endpoints/[id]/route.ts

```
@@ -0,0 +1,440 @@
+        }
+        break
+
+      case 'email':
+        if (!config.forwardTo) {
+          return { valid: false, error: 'Forward-to email address is required' }
+        }
+        if (typeof config.forwardTo !== 'string') {
+          return { valid: false, error: 'Forward-to email must be a string' }
+        }
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+        if (!emailRegex.test(config.forwardTo)) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)